src/App.js

```diff
@@ -0,0 +1,29 @@
+import './App.css';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { fetchPlanets } from './redux/home/homeSlice';
+import Home from './components/Home';
+import Details from './components/Details';
+
+const App = () => {
+  const refresh = useSelector((state) => state.planets);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchPlanets());
+  }, [dispatch, refresh.refresh]);
+
+  return (
+    <BrowserRouter>
+      <>
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/Details" element={<Details />} />
```

Comment:
- [ ]  For showing the specific planet list the route URL depends on the planet name or some unique key.For ex:-
`<Route exact path="/item/:id" element={Item} />`
Here id is a unique key for specific items, same you can show the planet name or planet_id as a unique key.
![Screenshot from 2023-01-14 01-42-29](https://user-images.githubusercontent.com/26767383/212411407-d397f7c4-b788-44a6-98f6-a948c29deb70.png)

Reply:
Thank you for reviewing my code. This part seems to be working for me just fine. It takes me to each and every unique planet.